README.adoc

```diff
@@ -285,6 +285,102 @@ spec:
+
+For Kiali an error request is when:
+
+* For http the code status match the expression */^[4-5]\d\d$/*
+* For grpc the code status match the expression */^ [1-9]$|^1[0-6]$/*
+
+=== Configuration
+
+Kiali works with a FIFO configuration, so Kiali will use the first configuration that applies with the resource requested.
+
+So the granularity and criteria should be configured from highest to lowest.
+
+The health configuration follow this scheme:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: string
+      kind: string
+      name: string
+      tolerance:
+        - protocol: string
+          direction: string
+          code: string
+          degraded: number
+          failure: number
+          ...
+    ...
+
+
+----
+
+For each Rate Kiali will use the first match by:
+
+* Namespace (string) is a regular expression that match the namespace name (bookinfo, istio-system...)
+* Kind (Optional) (string) is a regular expression that match the resource type (app/workload/service)
+* Name (Optional) (string) is a regular expression that match the resource name (reviews, productpage...)
+
+If a configuration match kiali will use the first tolerance configuration that match:
+
+* Request protocol is a plain string (http/grpc...)
+* Direction (string) is a regular expression that match the request (inbound/outbound)
+* Code (string) is a regular expression that match for each request the status_code to calculate the Error Rate.
+* Degraded is the threshold to show a *"degraded"* state for a resource
+* Failure ios the threshold to show a *"failure"* state for a resource
+
+
+Example configuration:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: "default"
+      tolerance:
+        - protocol: "http"
+          direction: "bound"
+          code: "400"
+          degraded: 5
+          failure: 10
+        - protocol: "grpc"
+          direction: "inbound"
+          code: "^1$"
+          degraded: 10
+          failure: 15
+    - namespace: "bookinfo"
+      kind: app
+      name: reviews
+      tolerance:
+        - protocol: "http"
+          direction: "outbound"
+          code: "400"
+          degraded: 1
+          failure: 5
+----
+
+In this case we created :
+
+*  For namespace default:
+  ** A "degraded" threshold in http protocol and if direction match the regular expression bound (In this case for inbound and outbound), when code mach the regular expression 400 rate is >=5% and failure alert if >=10%
+  ** A "degraded" threshold in grpc protocol and if direction match the regular expression inbound (In this case for inbound), when code mach the regular expression 1 rate is >=10% and failure alert if >=15%
+* For namespace bookinfo, app reviews:
+  ** A "degraded" threshold in http protocol and if direction match the regular expression outbound (In this case for outbound and outbound), when code mach the regular expression 400 rate is >=1% and failure alert if >=5%
+
+
+
+
```

Comment:
It's good to have documentation.
I've added some comments as this feature is new and it needs additional description to understand what it does (it requires a lot of implicit knowledge).

I don't think the README.adoc is the proper place for this.
Better to add a section in the Features where we are consolidating things in the kiali.io.